### PR TITLE
chore(docs): Updates Title Case and pronouns

### PR DESCRIPTION
**Pull request by SarahLKatz**
## Description

Adds correct title case and you/your pronouns to 4 files

## Related Issues

Addresses #18284

### CODE REVIEW

File: docs/docs/creating-a-transformer-plugin.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Just like a source plugin, a transformer plugin is a normal NPM package. It has
 
 `gatsby-transformer-yaml` is transformer plugin that looks for new nodes with a media type of text/yaml (e.g. a .yaml file) and creates new YAML child node(s) by parsing the YAML source into JavaScript objects.
 
-As an example, let's partially rebuild a simplified `gatsby-transformer-yaml` directly in an example site. Let's say we have a default gatsby starter site which includes a `src/data/example.yml` file:
+Checkout this example of rebuilding a simplified `gatsby-transformer-yaml` directly in an example site. For this example, you have aa default gatsby starter site which includes a `src/data/example.yml` file:
```
Comment (marcysutton) [inline docs/docs/creating-a-transformer-plugin.md]:
```suggestion
Check out this example of rebuilding a simplified `gatsby-transformer-yaml` directly in a site. Say you have a default Gatsby starter site which includes a `src/data/example.yml` file:
```

File: docs/docs/creating-prefixed-404-pages-for-different-languages.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ title: Creating Prefixed 404 Pages for Different Languages
 
 Using the [`onCreatePage`](/docs/node-apis/#onCreatePage) API in your project's `gatsby-node.js` file, it's possible to create different 404 pages for different URL prefixes, such as `/en/`).
 
-In the following example, we will create an English 404 page at `src/pages/en/404.js`, and a German 404 page at `/src/pages/de/404.js`. Here is a simple example:
+Here is an exampe that shows you how to create an English 404 page at `src/pages/en/404.js`, and a German 404 page at `/src/pages/de/404.js`:
```
Comment (marcysutton) [inline docs/docs/creating-prefixed-404-pages-for-different-languages.md]:
```suggestion
Here is an example that shows you how to create an English 404 page at `src/pages/en/404.js`, and a German 404 page at `/src/pages/de/404.js`:
```

Comment (SarahLKatz) [reply docs/docs/creating-prefixed-404-pages-for-different-languages.md]: Thank you for catching this!

File: docs/docs/creating-a-transformer-plugin.md (hunk 1 above)
Comment (SarahLKatz) [reply docs/docs/creating-a-transformer-plugin.md]: This is a great change - thank you! I wasn't sure the best way to simplify it - your suggestion is fantastic.

**Pull request merged**